engine/runtime/src/main/scala/org/enso/compiler/phase/ImportResolver.scala

```diff
@@ -109,6 +109,13 @@ class ImportResolver(compiler: Compiler) {
                 val converted = bindings
                   .toConcrete(compiler.packageRepository.getModuleMap)
                   .map { concreteBindings =>
+                    compiler.context.updateModule(
+                      current,
+                      { u =>
+                        u.bindingsMap(concreteBindings)
+                        u.loadedFromCache(true)
```

Comment:
Should we maybe have more than 2 states loaded/not yet loaded? It feels like 1) `not yet loaded` 2) `loaded only bindings` 3) `full IR loaded`

Reply:
There is nobody reading the "three states" yet. Why shall we keep three states in memory when we only read two?